lms/djangoapps/completion/waffle.py

```diff
@@ -83,3 +87,22 @@ def visual_progress_enabled(course_key):
         return waffle_flag()[ENABLE_COURSE_VISUAL_PROGRESS].is_enabled(course_key)
 
     return True
+
+
+def new_course_outline_enabled(course_key):
+    """
+    Returns whether the new course outline is enabled,
+    which requires visual progress to be enabled, as well.
+
+    :return:
+        bool -> True if visual progress is enabled
+        and the new course outline is enabled
+    """
+    return waffle().is_enabled(ENABLE_NEW_COURSE_OUTLINE) and visual_progress_enabled(course_key)
+
+
+def site_configuration_enabled():
```

Comment:
I think you have to rebase your branch, Greg changed/deleted this last week (it's a function that doesn't do anything).